src/shoobx/wfmc/process.py

```diff
@@ -51,6 +51,17 @@ def defaultDeadlineCanceller(process, deadline):
         deadline.deadlineTimer.cancel()
 
 
+def getInitialDataFieldsValues(datafields, evaluator):
```

Comment:
This probably also needs `strict` parameter, similar to `evaluateInputs()`?

Reply:
I was following same logic as it was done for `process` `start` method. But I guess it makes sense to normalize it if initial value can't be evaled. So for `start` method I assume it should use strict mode? 
Maybe then I should add some helper function that handles evaluation? Because if both need it, we should reuse it, instead of copy/pasting same logic:)

Reply:
I guess you can add a helper function if you think it is worth it (I don't think you will save a with it lot though).